packages/replay-next/components/console/filters/EventsList.tsx

```diff
@@ -84,7 +83,9 @@ function EventsListCategories({
           filterByText={filterByText}
         />
       ))}
-      <div className={styles.Header}>Other Events</div>
+      {commonEventCategories.length && otherEventCategories.length && (
+        <hr className="border-splitter" />
+      )}
```

Comment:
Removing these category headers looks like a UI change that's maybe out of scope of the "Chromium events" change?

cc @jonbell-lot23

Reply:
Thanks for letting me know! I need to track changes like this, so it's very helpful to be pinged.

In this case, we talked about making that change many weeks ago and I thought it was in prod already. I agree it's not related to this ticket, but when we're ready for it to go live, it has a thumbs up from design.

Reply:
I reached out to Jon back then, and he remembers after all this time ♥